### DISCUSSION TITLE
Bilardo topları toplama sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#98] [Zeka Sorusu] [answered]
![8vgruftonbuu](https://github.com/sonsuzus/sonsuzus/assets/52765796/679e1345-3c16-4e92-aaac-be511d25042c)

Boşluklara hangi toplar konmalıdır

**Comment by Berkek007** [comment, marked as answer]
Cavap: 13,11 ve 6. Toplamı 30 ediyor. 6 sayısı burada yok gibi görünse de 9 yazılı topu döndürdüğümüzde 6 sayısını elde diyoruz. Böylece 13+11+6= 30 u buluyoruz.
